src/main/java/com/contrastsecurity/task/VerifyThresholdsTask.java

```diff
@@ -90,7 +91,10 @@ public TaskResult execute(@NotNull final TaskContext taskContext) throws TaskExc
         try {
             //Get app and server id
             String applicationId = getApplicationId(contrast, profile.getUuid(), app_name);
-            long serverId = getServerId(contrast, profile.getUuid(), server_name, applicationId);
+            long serverId = 0;
```

Comment:
Can you switch this to null? I know `0` isn't a valid id for us, but it could be a valid id for something else.